espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/SetClipboardParams.kt

```diff
@@ -16,8 +16,23 @@
 
 package io.appium.espressoserver.lib.model
 
+import com.google.gson.annotations.SerializedName
+
 data class SetClipboardParams(
-    val contentType: ClipboardDataType = ClipboardDataType.PLAINTEXT,
+    @SerializedName("contentType")
+    private val _contentType: String?,
```

Comment:
what the point in having the underscore in the property name?

Reply:
To avoid to conflict with `val contentType : ClipboardDataType` in line 26